### ISSUE TITLE
 Unused variable 'k' 

### ISSUE THREAD

**Issue by BSravanthi** [virtual-labs/artificial-neural-networks-iiith#59]
Having the unused variable in a project can increase the maintenance cost, proliferating dead code and increasing the cognitive burden of the programmer.
**Please refer to the following link to fix similar issues.**
https://app.codacy.com/app/BSravanthi/artificial-neural-networks-iiith/issues?&filters=W3siaWQiOiJMYW5ndWFnZSIsInZhbHVlcyI6W251bGxdfSx7ImlkIjoiQ2F0ZWdvcnkiLCJ2YWx1ZXMiOlsiVW51c2VkIENvZGUiXX0seyJpZCI6IkxldmVsIiwidmFsdWVzIjpbbnVsbF19LHsiaWQiOiJQYXR0ZXJuIiwidmFsdWVzIjpbMTcyN119LHsidmFsdWVzIjpbXX1d

**Comment by DevikaBoddu** [comment]
Commit id : c2dd9ed(exp1)
 Commit id : 55b5c13(exp10)
 Commit id : 49428d2(exp2)
 Commit id : fd2ae6f(exp3)
 Commit id : 9e034e9(exp4)
 Commit id : d034d01(exp5)
 Commit id : 665689c(exp6)
 Commit id : aeacaf9(exp7)
 Commit id : c458c83(exp8)
 Commit id : 821637d(exp9)
 Commit id : f75fada(scrap)

**Comment by sadhusantosh1999** [comment]
valid fix: @DevikaBoddu

**Comment by AksharaMayreddy** [comment]
valid fix @DevikaBoddu

**Comment by abhishek5460** [comment]
Not valid fix: @DevikaBoddu

**Comment by abhishek5460** [comment]
Commit id's:
ee93081cef64612fedcf6f60f89712204d88d9f3(exp1)
e6e4e85ec8a78c53e3549a18364987e1541190ba(exp2)
3ad613cffed52c5177a0b9506e5cd83fe12040b3(exp3)
78aa934bee5d325d5ee7fa00db29594fcefe7c5e(exp4)
9b16310fcc65d4f42f4077ff20f92bbf1071aa6a(exp5)
6735f072dd4ee6686dfe31340825fd87ce7f67e8(exp6)
98ec5dfbded5a17dc1f9238701c84141f233c109(exp7)
18ddc136216060660408716789809ba213471e01(exp8)
9b15276cce2d5822858c6424629fd57eb82dce0e(exp9)
497e419bde46ea003da4c1c3e4e6db562bf547a2(exp10)
c882439902ec916695f0cf01bb4489e85df5d6cf(lab)

**Comment by DevikaBoddu** [comment]
No,l have removed the unused variables that 're mentioned so it can be validated,You have removed the unused variables that aren't mentioned too. @abhishek5460
